Src/FluentAssertions/AssertionExtensions.cs

```diff
@@ -294,7 +294,7 @@ public static StringCollectionAssertions Should(this IEnumerable<string> @this)
         /// current <see cref="IDictionary{TKey, TValue}"/>.
```

Comment:
Most of the XML docs still refer to `IDictionary`

Reply:
Fixed